include/ttmlir/Dialect/TTNN/IR/TTNNOps.td

```diff
@@ -784,6 +795,14 @@ def TTNN_EmptyOp : TTNN_Op<"empty", [NoMemoryEffect]> {
                          OptionalAttr<TTNN_MemoryConfigAttr>:$memory_config);
     let results = (outs AnyRankedTensor:$result);
 
+    let extraClassDeclaration = [{
+      wa::TTNNOperandsWorkarounds getOperandsWorkarounds() {
+        wa::TTNNOperandWorkarounds rowMajorLayoutWorkaround = wa::TTNNOperandWorkarounds(Layout::RowMajor);
+        return wa::TTNNOperandsWorkarounds::createEmptyTTNNOperandsWorkarounds()
+        .addOutputOperandWorkaround(rowMajorLayoutWorkaround);
```

Comment:
I agree, it can simplify things further, but I wouldn't consider this for now, as Layout(and other workaround attributes) doesn't represent OperandWorkarounds, hence addOutputOperandWorkaround will become a bit strange...